__tests__/app.test.js

```diff
@@ -43,7 +45,41 @@ describe.only('api/topics', () => {
                 const message = { msg: "path not found"};
                 expect(response.body).toEqual(message)
             })
-            
-        })
+        });
+    });
+    describe('/api/articles/:article_id', () => {
+        test('status 200 - responds with a specific article object', () => {
+            return request(app)
+            .get(`/api/articles/1`)
+            .expect(200)
+            .then((response) => {
+               
+                expect(response.body.article).toEqual(expect.objectContaining({
+                    article_id: expect.any(Number),
```

Comment:
I would say because we are testing specifically for article_id 1, we should expect to see that in the test, as opposed to any number